docs/chapters/architecture/Lifecycle/index.rst

```diff
@@ -0,0 +1,170 @@
+:orphan:
+
+.. _Life cycle-Management:
```

Comment:
The hyphen has ended up in the wrong place, in my opinion. It should be "Lifecycle Management" or "Life-cycle Management".

Reply:
And if it is changed here the same way of writing should be used throughout the documentation.